environment_gcc4.yml

```diff
@@ -1,4 +1,5 @@
 channels:
+  - krodriguez 
```

Comment:
I can upload the geos build to me channel if you can send me the tarball.

Reply:
sure :+1: